lib/jazzy/sourcekitten.rb

```diff
@@ -339,7 +339,7 @@ def self.make_source_declarations(docs, parent = nil)
         declaration.parent_in_code = parent
         declaration.type = SourceDeclaration::Type.new(doc['key.kind'])
         declaration.typename = doc['key.typename']
-        current_mark = SourceMark.new(doc['key.name']) if (declaration.type.mark? && (doc['key.name'].start_with?('MARK: ')))
+        current_mark = SourceMark.new(doc['key.name']) if (declaration.type.mark? && !(doc['key.name'].start_with?('TODO: ')))
```

Comment:
To me, the earlier sourcekitten.rb patch looks a bit better than this one because it handles 'FIXME' as well as 'TODO'.

CI failed with the earlier patch because objc *#pragma mark something* lines come in to jazzy as type.marks as well, with their key.name being just the *something* part, no preceding 'mark'.

I think this patch misbehaves for pathological objc like `#pragma mark TODO: finish these methods` 😄 

The logic needed for current_mark seems to be "objc || (swift beginning 'MARK: ')".  A neat fix might be to change `SourceDeclaration::Type.mark?` to take a parameter of the key.name and do the test there -- the code here in `mark?` knows whether it is from an objc-pragma or not -- 2 calls to mark?() in sourcekitten.rb to update.